src/main/java/com/checkmarx/flow/service/BitBucketService.java

```diff
@@ -245,18 +254,64 @@ private void sendCommitComment(ScanRequest request, String comment){
         restTemplate.exchange(request.getMergeNoteUri(), HttpMethod.POST, httpEntity, String.class);
     }
 
-    private static JSONObject getJSONComment(String comment) throws JSONException {
+    private static JSONObject getJSONComment(String comment) {
         JSONObject requestBody = new JSONObject();
         JSONObject content = new JSONObject();
         content.put("raw", comment);
         requestBody.put("content", content);
         return requestBody;
     }
 
-    private static JSONObject getServerJSONComment(String comment) throws JSONException {
+    private static JSONObject getServerJSONComment(String comment) {
         JSONObject requestBody = new JSONObject();
         requestBody.put("text", comment);
         return requestBody;
     }
 
+    @Override
+    public Sources getRepoContent(ScanRequest request) {
+        return null;
+    }
+
+    @Override
+    public CxConfig getCxConfigOverride(ScanRequest request) {
+        CxConfig result = null;
+        if (StringUtils.isNotBlank(properties.getConfigAsCode())) {
+            try {
+                result = loadCxConfigFromBitbucket(properties.getConfigAsCode(), request);
+            } catch (NullPointerException e) {
+                log.warn(CONTENT_NOT_FOUND_IN_RESPONSE);
+            } catch (HttpClientErrorException.NotFound e) {
+                log.info(String.format("No Config As code was found : %s", properties.getConfigAsCode()));
+            } catch (Exception e) {
+                log.error(ExceptionUtils.getRootCauseMessage(e));
+            }
+        }
+        return result;
+    }
+
+    private CxConfig loadCxConfigFromBitbucket(String filename, ScanRequest request) {
+        HttpHeaders headers = createAuthHeaders();
+        String repoSelfUrl = request.getAdditionalMetadata("repo-self-url");
+        String urlTemplate = repoSelfUrl.concat(FILE_CONTENT);
+        ResponseEntity<String> response = restTemplate.exchange(
+                urlTemplate,
+                HttpMethod.GET,
+                new HttpEntity<>(headers),
+                String.class,
+                request.getHash(),
+                filename
+        );
+        if (response.getBody() == null) {
+            log.warn(HTTP_BODY_IS_NULL);
+            return null;
```

Comment:
It's better to have a single return point instead of 3.

Reply:
Have done the changes. Please check if it looks okay.